doc/releases/release-notes-3.5.rst

```diff
@@ -100,14 +100,132 @@ Bluetooth
+  * Refactored bt_audio_codec_cfg to flat arrays
+  * Renamed codec qos framing
+  * Added API to get MICP service ATT handles
+  * Replaced BT_AUDIO_CODEC_LC3_ID -> BT_HCI_CODING_FORMAT_LC3
+  * Modify the ASCS ACL disconnect behavior
+  * Replaced the BT_AUDIO_CODEC_PARSE_ERR_ values with errno values.
+  * Added support for long read/write in CAP
+  * Removed CONFIG_BT_PACS_{SNK,SRC}_CONTEXT
+  * Added bt_audio_codec_cap_get helper functions
+  * Fixed issue with bt_iso_cig_reconfigure
+  * Added API to get VCP service ATT handles
+  * Split bt_audio_codec_meta_get to cfg and cap
+  * Fixed ASCS Source ASE link loss state transition
+
 * Direction Finding
 
 * Host
 
+  * Re-order LE L2CAP connected callback to be after connection response
+  * Fix to invalidate the Resolvable Private Address when starting legacy adv
+  * Introduced new API to manage Bluetooth settings storage
+  * Fixed a connection reference leak
+  * Fixed to abort pairing if connection disconnected
+  * Added SMP bondable flag overlay per connection
+  * Introduce BT_CONN_PARAM_ANY, allows setting any value to connection parameters
+  * Fix possible buffer overflow in bt_conn_le_start_encryption
+  * Added USE_NRPA advertising option
+  * Use RPA as PAwR responder address if BT_PRIVACY=y
+  * Fixed HCI ISO SDU length sent to controller
+  * Fixed some SMP issues
+  * Fixed OTS memory leak while procedure is not finished
+  * Amended existing L2CAP accept callbacks
+  * Fixed HCI ISO Data packets fragmentation
+  * Fixed forced pairing request handling
+
 * Mesh
 
+  * Added storing Private GATT Proxy state persistently
+  * Fixed Proxy Privacy parameter support
+  * Added support to use both tinycrypt and PSA based crypto
+  * Implementation to not poll friend when sending over loopback
+  * Added full virtual addresses support
+  * Added option to disable Label UUIDs recovery
+  * Added tf-m support
+  * Fixed the old internal encryption api usage for DFU metadata
+  * Fixed provisionee public key usage
+  * Added statistic module
+  * Fixed OOB info and URI hash in scan report message
+  * Updated default values for transport SAR configuration
+  * Align handling of seg_ack to Mesh Specification
+  * Fixed OP_AGG_SRV dependency of OP_AGG_CLI
+  * Use NRPA for private proxy advertisements
+  * Keep Composition Data Page 128 if RPR Server is present
+  * Fixed usage of Identity and NRPA adv options for proxy
+  * Added ext timeout in PB for OOB I/O
+  * Added support for comp data page 2
+  * Added support for Upload OOB Start
+  * Fixed static oob setting
+  * Fixed issue with RPR server and client.
+  * Fixed Upload Progress for already received fw
+  * Fixed Fw Dist Upload OOB Start msg length check
+  * Fixed to release previously reserved slot before reserving again
+  * Fixes for Coverity reported issues
+
 * Controller
 
+  * Fixed use of pre-programmed PPI on nRF SoCs
+  * Moved the EVENT_OVERHEAD_START_US verbose assertion to each state/role LLL
```

Comment:
do you really want/need to go to this level of detail? (this probably applies to several other entries but this is one that caught my eye)

Reply:
Do we have any guidelines of what goes into the release notes, and what does?

I assume anything that changes behavior, API or bug fixes